frontend/src/components/shared/Breadcrumb.tsx

```
@@ -0,0 +1,83 @@
+import Link from "next/link";
+import { IconHome } from "@tabler/icons-react";
+import {
+    Breadcrumb as BreadcrumbRoot,
+    BreadcrumbList,
+    BreadcrumbItem,
+    BreadcrumbLink,
+    BreadcrumbPage,
+    BreadcrumbSeparator,
+} from "@/components/ui/breadcrumb";
+
+interface BreadcrumbItem {
```

Biome (2.3.13)

[error] Shouldn't redeclare 'BreadcrumbItem'. Consider to delete it or rename it.

'BreadcrumbItem' is defined here:


(lint/suspicious/noRedeclare)